bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -112,6 +118,93 @@ async def autoreview_status(self, ctx: Context) -> None:
+        old_nominations = await self._get_forgotten_nominations()
+        untracked_nominations = await self._find_untracked_nominations(old_nominations)
+        for nomination, thread in untracked_nominations:
+            issue_created = await self._track_vote_in_github(nomination, thread.jump_url)
+            if issue_created:
+                await thread.starter_message.add_reaction(FLAG_EMOJI)
+
+    async def _get_forgotten_nominations(self) -> list[Nomination]:
+        """Get active nominations that are more than 2 weeks old."""
+        now = arrow.utcnow()
+        nominations = [
+            nomination
+            for nomination in await self.api.get_nominations(active=True)
+            if (now - nomination.inserted_at).days >= OLD_NOMINATIONS_THRESHOLD_IN_DAYS
+        ]
+        return nominations
+
+    async def _find_untracked_nominations(
+            self,
+            nominations: list[Nomination]
+    ) -> list[Tuple[Nomination, discord.Thread]]:
+        """
+        Returns a list of tuples representing a nomination and its vote message.
+
+        All active nominations are iterated over to identify whether they're tracked or not
+        by checking whether the nomination message has the "🎫" emoji or not
+        """
+        untracked_nominations = []
+
+        for nomination in nominations:
+            # We avoid the scenario of this task run & nomination created at the same time
+            if not nomination.thread_id:
+                continue
+
+            try:
+                thread = await get_or_fetch_channel(nomination.thread_id)
+            except discord.NotFound:
+                log.debug(f"Couldn't find thread {nomination.thread_id}")
+                continue
+
+            starter_message = thread.starter_message
+            if not starter_message:
+                # Starter message will be null if it's not cached
+                try:
+                    starter_message = await self.bot.get_channel(Channels.nomination_voting).fetch_message(thread.id)
+                except discord.NotFound:
+                    log.debug(f"Couldn't find message {thread.id} in channel: {Channels.nomination_voting}")
+                    continue
+
+            if FLAG_EMOJI in [reaction.emoji for reaction in starter_message.reactions]:
+                # Nomination has been already tracked in GitHub
+                continue
+
+            untracked_nominations.append((nomination, thread))
+        return untracked_nominations
+
+    async def _track_vote_in_github(self, nomination: Nomination, vote_jump_url: str | None = None) -> bool:
+        """
+        Adds an issue in GitHub to track dormant vote.
+
+        Returns True when the issue has been created, False otherwise.
+        """
+        if not GithubAdminRepo.token:
+            log.warning(f"No token for the {GithubAdminRepo.name} repository was provided, skipping issue creation.")
+            return False
```

Comment:
this check can be done in the init function so that the loop isn't even started if this secret set

Reply:
Done in 33415daa